content/en/docs/concepts/cluster-administration/addons.md

```diff
@@ -79,6 +79,8 @@ installation instructions. The list does not try to be exhaustive.
   Pods and non-Kubernetes environments with visibility and security monitoring.
 * [Romana](https://github.com/romana) is a Layer 3 networking solution for pod
   networks that also supports the [NetworkPolicy](/docs/concepts/services-networking/network-policies/) API.
+* [Spiderpool](https://github.com/spidernet-io/spiderpool) is an underlay and RDMA
```

Comment:
From the Spiderpool project's [README](https://github.com/spidernet-io/spiderpool?tab=readme-ov-file):
Spiderpool is an underlay and RDMA networking solution for Kubernetes. Spiderpool is supported on bare metal, virtual machines, and public cloud environments.

Reply:
Let's keep the text consistent with README.

Reply:
updated